package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@totallymoney/github-serverless-dotnet-artifacts",
-  "version": "2.0.1",
```

Comment:
we don't need to bump the version manually..

we just do these steps after the PR is merged..
https://github.com/totallymoney/github-serverless-dotnet-artifacts#development